HeartbeatKit/src/initialization.js

```diff
@@ -19,34 +20,50 @@ var Initialization = {
             var adobeHeartbeatSdk = document.createElement('script');
             adobeHeartbeatSdk.type = 'text/javascript';
             adobeHeartbeatSdk.async = true;
-            adobeHeartbeatSdk.src = 'https://static.mparticle.com/sdk/web/adobe/MediaSDK.min.js';
+            adobeHeartbeatSdk.src =
+                'https://static.mparticle.com/sdk/web/adobe/MediaSDK.min.js';
             (
                 document.getElementsByTagName('head')[0] ||
                 document.getElementsByTagName('body')[0]
             ).appendChild(adobeHeartbeatSdk);
             adobeHeartbeatSdk.onload = function() {
-                if (ADB && eventQueue.length > 0) {
-                    // Process any events that may have been queued up while forwarder was being initialized.
-                    for (var i = 0; i < eventQueue.length; i++) {
-                        processEvent(eventQueue[i]);
+                if (ADB) {
+                    isHBInit = self.initHeartbeat(
```

Comment:
i wonder if there's a better way of doing this.

`self.initHeartBeat` calls a callback which sets `isHBInit`, but here, we make `isHBInit` equal to the result of `self.initHeartBeat`. If `self.initHeartbeat` doesn't return false on line 113, is there any chance that the callback sets `isHBInit = true`, but because self.initHeatbeat doesn't return anything, isHBInit is then set to `undefined`?